### DISCUSSION TITLE
Got "Not found" with d1-mini sp8266 dev-board from berrybase

### DISCUSSION THREAD

**Discussion by sorryusernameisalreadytaken** [TheRealWaldo/esp8266-amg8833#10] [Q&A]
Hey there.

Because I want to use the normal git-master state without additional changes (setup ports manually in code), what does not work with my [Goouuu Mini-S1](https://github.com/TheRealWaldo/esp8266-amg8833/discussions/8), I ordered d1-mini esp8266 dev-board from [berrybase](https://www.berrybase.de/d1-mini-esp8266-entwicklungsboard).

I just checkout the latest git state and run VS-Code and let it flash to the board. Then I add my WIFI-Data. After that I got always: "Not found" error.
I connect SDA to D2 and SCL to D1. GND to GND and VIN to 3V3.

Any idea?

**Comment by TheRealWaldo** [comment]
I am unfamiliar with that board.

Can you provide additional information, such as log output and what is giving you the "not found" error?

**Reply by sorryusernameisalreadytaken** [reply]
Sorry. I got the "Not found" in the html web-page of the esp8266 itself after init the wifi-daten and let it get a ip from my dhcp.

Where can I see the log output in vscode?

**Reply by TheRealWaldo** [reply]
It doesn't have a UI or any content at the root.

The JSON which the Thermal application consumes is exposed at `/raw`
